plugins/ui-kit-candidates/SortableColumnHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { SortableColumnHeaderCellIcon } from "#PLUGINS/ui-kit-candidates/SortableColumnHeaderCellIcon";
+
+type SortDirection = "ASC" | "DESC";
+
+export function SortableColumnHeader({
+  sortHandler,
+  sortColumn,
+  sortDirection,
+  columnName,
+  columnTitle
+}: {
+  sortHandler: (columnName: string) => void;
+  sortColumn: string;
+  sortDirection: SortDirection;
+  columnName: string;
```

Comment:
Let's reduce the amount of props by partial applying the sort handler and adjusting the sort direction to include "INACTIVE" (`undefined`) as discussed.